tests/rptest/services/redpanda.py

```diff
@@ -3732,7 +3732,7 @@ def is_allowed_log_line(line: str) -> bool:
                     continue
 
                 if is_allowed_log_line(line):
-                    self.logger.warn(
+                    self.logger.info(
```

Comment:
I am usually a big fan of this kind of changes however this will make oversized alloc warnings which are not fatal less likely to be seen. 

I guess the real fix for this is to reduce the fatal limit in DT down from 400KiB to something more realistic but we should probably do that first before changing the logger level.

Reply:
AFAICT those use a different log line? See [self.logger.warn(](https://github.com/redpanda-data/redpanda/blob/86741ac412203439e68f5ff1f9e854df036e7249/tests/rptest/services/utils.py#L165).

Reply:
Ok great, this feels like just a repeat of the discussion we had the other day. There is too many places that log checking.

Reply:
Yeah, I guess there is some valid reason for this case though since we are parsing the log line and applying our specific threshold to it. Once we get rid of that threshold, this logic can disapear and then the test will simply fail by the usual checking if it does a large alloc.